mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/195 - property panel missing properties

Closes #195 

To test:
- Run different scenarios and check the properties in the issue to make sure they look correct
